Sources/TensorFlow/Layers/Convolutional.swift

```diff
@@ -641,3 +641,98 @@ public extension DepthwiseConv2D {
             padding: padding)
     }
 }
+
+/// A layer for adding Zero-padding in the temporal dimension.
+public struct ZeroPadding1D<Scalar: TensorFlowFloatingPoint>: Layer {
+    /// The padding values along the temporal dimension.
+    @noDerivative public let padding: (Int, Int)
+
+    /// Creates a Zero-padding 1D Layer.
+    ///
+    /// - Parameter padding : A tuple of two integers describing how many zeros to be padded at the
```

Comment:
```suggestion
    /// - Parameter padding: A tuple of two integers describing how many zeros to be padded at the
```
Same everywhere else.